dipy/workflows/viz.py

```diff
@@ -140,6 +148,9 @@ def run(self, input_files, cluster=False, cluster_thr=15.,
         emergency_ref = create_nifti_header(hdr['affine'], hdr['dims'],
                                             hdr['vox_size'])
 
+        if interactive:
+            self.manage_output_overwrite = self.manage_output_overwrite_helper
+            
```

Comment:
@skoudoro, this part of the commit makes sure the change is effective only in interactive mode.